### ISSUE TITLE
Disabling "Allow frontend edit" hides registration input

### ISSUE THREAD

**Issue by muhpixel** [diglin/Diglin_Username2#22]
I was wondering why this would be. The comment for the "Allow frontend edit" clearly states it's for making it editable in the user's account page, and this is how the Magento 1 version of this plugin works.  However, I installed it today on Magento 2 (this was my first 3rd party module installed on Magento 2), and I noticed the register.phtml even has a condition for this:

` if ($_username->isEnabled() && $_username->isEditable()):`

So, having it disabled completely blocks it from being able to be set at registration. I'm not sure if this was intended, and it would be easy for me to adjust it if this was the intention. But was it?

**Comment by sylvainraye** [comment]
Hi @muhpixel ,

Thank you for your feedback.

Well, I agree it could be disappointing. I think the best should be to habe a separate configuration option to allow to create a username at registration or not. Completely separated from editing from user account.

What do you think ?

**Comment by muhpixel** [comment]
@sylvainraye Definitely. I know many sites that use a username type of community, prefer that most users choose their own username when they register, and then they do not wish for them to be able to edit them on their own afterwards.
